src/protocols/rdp/guac_rdpdr/rdpdr_printer.c

```diff
@@ -141,18 +141,20 @@ static void guac_rdpdr_device_printer_announce_handler(guac_rdpdr_device* device
     Stream_Write(output_stream, "PRN1\0\0\0\0", 8); /* DOS name */
 
     /* Printer data */
-    Stream_Write_UINT32(output_stream, 24 + GUAC_PRINTER_DRIVER_LENGTH + GUAC_PRINTER_NAME_LENGTH);
+    char* utf16_printer_name = malloc(1);
```

Comment:
As with comment above on `realloc()`, this feels a little kludgy, but allows for dynamic allocation to work.

Reply:
I am definitely not a fan of dynamically allocating one byte just so we can `realloc()` it immediately after. I suspect we don't actually need to do this though - see below.

Reply:
Removed.